HelpSource/Guides/GUI-Introduction.schelp

```diff
@@ -5,41 +5,31 @@ categories:: GUI
 note::
 To avoid confusion, we start off with an important historical note. SuperCollider used to have a pretty nasty GUI class system that involved platform-dependent classes like SCWindow. This system is completely gone in 3.7 with a unified Qt GUI.
 
-You will still see remnants of the old GUI classes in some SuperCollider help files and quarks. We're still working on updating all the help files. If you encounter such old classes in a quark, you can do a public service by informing the quark author.
+You will still see remnants of the old GUI classes in some SuperCollider help files, examples, and quarks. We're still working on updating all the help files. If you encounter such old classes in a quark, you can do a public service by informing the quark author.
 ::
 
-SECTION:: Basic elements: Windows, views and containers
+SECTION:: Views
 
-The most fundamental element of the GUI is the strong::Window::. It occupies a rectangular space on screen within which other GUI elements are displayed. It usually has a bar that displays the window's title and allows for moving it, resizing it and closing it with the controls it displays or through mouse and keyboard interaction. Some of these aspects may be controlled within SuperCollider GUI code, though it is largely platform-dependent how precisely interaction with a window happens and is visually indicated.
+The most fundamental element of the GUI is the strong::view::. They all inherit from the basic View class. The view occupies a rectangular space on screen within which it draws itself to display some data or to indicate a mode of interaction between the user and the program. Views receive keyboard and mouse events generated by the user and respond to them by controlling the behavior of the program. They also display information about the state of the program and the data on which it operates.
```

Comment:
Yes but my point is, I don't think that the *Gui introduction* help file should emphasise too much on the `View`class. Maybe further down in something like *Writing your own GUI classes*?

I'm a bit sceptical when I see users posting code with the old `*.addFlowLayout()` way when using Qt layouts is so much easier.
Again, my 2 cents..